### PR TITLE
replace old explorer url with the new one and remove edit option

### LINKED ISSUE #1766: Explorer - URL tied to profile
Currently, when we change the URL for explorer for old profiles old URL stays.

Why it's saved on profile level? To allow edit it? @coiki do you think anyone is using this option?

My propositions:
- remove the possibility to edit explore link **or**
- provide versioning. If the version increases then overwrite URL stored on profile level with current one.

image

@coiki what you think about that?

### PR DESCRIPTION
**Pull request by bassemmagdy**
-fixes  #1766

### CODE REVIEW

File: src/core/database/storage/SettingsModelStorage.ts [hunk 1]
```diff
@@ -84,6 +84,10 @@ export class SettingsModelStorage extends VersionedObjectStorage<Record<string,
                     description: 'Reset for fees update.',
                     migrate: () => undefined,
                 },
+                {
```
Comment (cryptoBeliever) [inline src/core/database/storage/SettingsModelStorage.ts:87]: - We need reset storage to remove explorer link from there?
- description is wrong

File: src/views/forms/FormGeneralSettings/FormGeneralSettings.vue [hunk 2]
```diff
@@ -42,6 +40,15 @@
                         </template>
                     </FormRow>
 
+                    <FormRow>
+                        <template v-slot:label> {{ $t('set_explorer_link') }}: </template>
```
Comment (OlegMakarenko) [inline src/views/forms/FormGeneralSettings/FormGeneralSettings.vue:44]: Maybe for consistency with the labels above, we should change it to this:
```suggestion
                        <template v-slot:label> {{ $t('form_label_explorer_link') }}: </template>
```
Then the translation key should be also updated

Comment (bassemmagdy) [reply src/views/forms/FormGeneralSettings/FormGeneralSettings.vue]: Updated.

**Comment by cryptoBeliever** [comment]
Tested. Works well.

**Review by cryptoBeliever — APPROVED** [review]

**Comment by bassemmagdy** [comment]
rebased in https://github.com/symbol/desktop-wallet/pull/1784

**Pull request closed without merging**